### PR TITLE
Remove deprecated "facts" aliases

### PR DESCRIPTION
**Pull request by jillr**
##### SUMMARY

Modules named "facts.py" that do not return ansible_facts were renamed to "info.py" in 2.9. Remove these aliases now that the deprecation period is over.

This PR should be included in 3.0.0 of the collection.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
*_facts.py

### CODE REVIEW

**Comment by jillr** [comment]
TODO:
- [ ] Test cloudfront_info (removed ansible_facts, no tests)
- [ ] Test ecs_service_info (removed ansible_facts, tests are disabled in CI)

**Review by tremble — CHANGES_REQUESTED** [review]
ansible-2.9 sanity ignores needs updating

Is it worth splitting out the two modules with missing tests rather than blocking the bulk of the cleanup?

**Comment by jillr** [comment]
@tremble I will hopefully get to them today, but I can totally do it under a separate PR

**Comment by jillr** [comment]
cloudfront_info and ecs_service_info tested successfully with the latest change

File: plugins/modules/cloudfront_info.py [hunk 1]
```diff
@@ -658,10 +636,8 @@ def main():
 
     result['changed'] = False
     result['cloudfront'].update(facts)
-    if is_old_facts:
-        module.exit_json(msg="Retrieved CloudFront facts.", ansible_facts=result)
-    else:
-        module.exit_json(msg="Retrieved CloudFront info.", **result)
+
+    module.exit_json(msg="Retrieved CloudFront facts.", **result)
```
Comment (tremble) [inline plugins/modules/cloudfront_info.py:640]:
```suggestion
    module.exit_json(msg="Retrieved CloudFront info.", **result)
```

File: plugins/modules/efs_info.py [hunk 2]
```diff
@@ -387,7 +382,6 @@ def main():
         targets = [(item, prefix_to_attr(item)) for item in targets]
         file_systems_info = [item for item in file_systems_info if has_targets(item['mount_targets'], targets)]
 
-    if is_old_facts:
         module.exit_json(changed=False, ansible_facts={'efs': file_systems_info})
```
Comment (tremble) [inline plugins/modules/efs_info.py:385]: this still appears to be returning a fact

**Review by tremble — CHANGES_REQUESTED** [review]
A couple of minor niggles

File: plugins/modules/efs_info.py (hunk 2 above)
Comment (jillr) [reply plugins/modules/efs_info.py:385]: /me headdesk - I actually made these changes then I stashed them while testing back and forth between main and this branch.  Thank you.  :smile:

**Review by tremble — APPROVED** [review]

**Pull request merged**